app/src/main/java/com/fastlink/zekrapp/appData/model/CategoryModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.fastlink.zekrapp.appData.model
+
+data class CategoryModel(
+    val id: Int,
+    val categoryTitle: String,
+    val plainCategoryTitle: String,
```

Comment:
What does plainCategoryTitle mean, and why is it here if it's not used?

Reply:
we use it but not now, plainCategoryTitle represents the category title without diacritical marks (i.e., without kasra, fatha, or other vowel signs) , if we add search functionality we use this plain Category title